### DISCUSSION TITLE
Maximum thrust

### DISCUSSION THREAD

**Discussion by fernandafenelon** [JSBSim-Team/jsbsim#565] [Q&A]
Hello everyone,

is there a variable that outputs the maximum thrust on a specified altitude and speed?

**Comment by seanmcleod** [comment]
Pretty sure there isn't. However you can get the information from the engine file, e.g. take a look at:

https://github.com/JSBSim-Team/jsbsim/blob/db4d967b8e441ce0406b0c5baaad3201294386a9/engine/CFM56.xml#L14-L15

https://github.com/JSBSim-Team/jsbsim/blob/db4d967b8e441ce0406b0c5baaad3201294386a9/engine/CFM56.xml#L42-L57
